pkgs/development/tools/misc/argbash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, autoconf }:
+
+stdenv.mkDerivation rec {
+  pname = "argbash";
+
+  version = "2.8.1";
+
+  src = fetchFromGitHub {
+    owner = "matejak";
+    repo = "argbash";
+    rev = "${version}";
+    sha256 = "0jzfr2w5pza33mvw6hq6rm1vjgrfy95dp4421403jnji3ziwgf0w";
+  };
+
+  sourceRoot = "${src}/resources";
+
+  nativeBuildInputs = [ autoconf ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "Bash argument parsing code generator";
+    homepage = "https://argbash.io/";
```

Comment:
No quote around the url.

Reply:
Please keep the quote around the url, as per https://github.com/NixOS/rfcs/pull/45.